conda/models/dist.py

```diff
@@ -17,12 +17,19 @@
 from ..common.constants import NULL
 from ..common.url import has_platform, is_url, join_url
 from ..deprecations import deprecated
+from ..history import dist_str_to_quad, strip_extension  # noqa: F401
 from .channel import Channel
 from .package_info import PackageInfo
 from .records import PackageRecord
 
 log = getLogger(__name__)
 
+deprecated.module(
+    "24.3",
+    "24.9",
+    addendum="Use `conda.models.records.PackageRecord` instead.",
```

Comment:
`PackageRecord` of course does not cover every functionality of this module one to one.
Feel free to add additional deprecation notes to symbols of this module as you see fit!